README.md

```diff
@@ -31,6 +31,20 @@ Alternatively, you can run **rebase-helper** in a container:
 
 See [docker reference](https://rebase-helper.readthedocs.io/en/latest/user_guide/rebasing_in_container.html) for more information.
 
+## What do I get from **rebase-helper**?
+
+**rebase-helper** creates *rebase-helper-results* directory which contains the following useful items:
```

Comment:
`useful` sounds a bit over the top to me. And what's wrong with `always`?

Reply:
I do not understand the question. What is wrong with what `always`?

Reply:
I was referring to my proposal in comment https://github.com/rebase-helper/rebase-helper/pull/477#issuecomment-380441060.